packages/gatsby-source-filesystem/README.md

```diff
@@ -41,6 +41,33 @@ module.exports = {
 };
 ```
 
+```javascript
+// In your gatsby-config.js
+module.exports = {
+  plugins: [
+    // The following setup creates "File" nodes for all files from
+    // "data" directory except files with "xml" extension.
+    //
+    // The "ignore" parameter must be one of the following:
+    //    - string to be directly matched,
```

Comment:
These are the options chokidar supports?

Perhaps we should highlight that what they add here will go directly into Chokidar?